assets/src/adaptivity/capi.ts

```diff
@@ -98,12 +99,70 @@ export interface ICapiVariableOptions {
   writeonly?: boolean;
   allowedValues?: string[];
   bindTo?: string;
+  shouldConvertNumbers?: boolean;
 }
 
 export interface JanusConditionProperties extends ConditionProperties {
   id: string;
   type?: CapiVariableTypes;
 }
+export const parseArrayWithStringConversion = (val: unknown): unknown[] => {
```

Comment:
Created a new function to handle this just for shake of minimal impact on any other areas.